bylaws/003-membership.md

```diff
@@ -98,9 +98,13 @@ will be assumed.
 
 If, in the judgement of PHP-FIG, a Voting Representative is acting
 inappropriately and to the detriment of PHP-FIG's ability to meet its
-objectives, a vote may be taken to request a replacement Voting Representative in
-accordance with the [Voting Protocol bylaw][voting] or to expel the Member
-Project where replacing a Voting Representative is not possible.
+objectives, a vote may be taken to request a replacement Voting Representative,
+or to expel the Member Project where replacing a Voting Representative is not
```

Comment:
Ah fair enough. We should make this clarification to L130 to clarify that member projects can be removed as the result of a direct vote as well. The point being made is that we should be able to expel projects without giving them the ability to provide a replacement representative as well as just being able to request a new representative.